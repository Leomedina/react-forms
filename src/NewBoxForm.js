import React from 'react';

const newBoxForm = () => (
  null
);

export default newBoxForm;