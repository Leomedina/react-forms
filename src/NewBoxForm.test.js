import React from "react";
import { render } from "@testing-library/react";
import NewBoxForm from './NewBoxForm';

describe("Smoke and snapshot tests", function () {
  it("should render without crashing", function () {
    const addBox = () => null;
    render(<NewBoxForm addBox={addBox} />);
  });

  it("should match snapshot", function () {
    const addBox = () => null;
    const { asFragment } = render(<NewBoxForm addBox={addBox} />);
    expect(asFragment()).toMatchSnapshot();
  });
});