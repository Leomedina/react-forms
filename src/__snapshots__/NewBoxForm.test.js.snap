// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Smoke and snapshot tests should match snapshot 1`] = `
<DocumentFragment>
  <form>
    <p>
      <label
        for="color"
      >
        color:
      </label>
      <input
        id="color"
        name="color"
        value=""
      />
    </p>
    <p>
      <label
        for="width"
      >
        width:
      </label>
      <input
        id="width"
        max="100"
        name="width"
        type="range"
        value="50"
      />
      <label
        for="height"
      >
        height:
      </label>
      <input
        id="height"
        max="100"
        name="height"
        type="range"
        value="50"
      />
    </p>
    <button>
      Add Box
    </button>
  </form>
</DocumentFragment>
`;
